packages/firestore/src/remote/backoff.ts

```diff
@@ -30,8 +30,17 @@ const LOG_TAG = 'ExponentialBackoff';
  */
 export class ExponentialBackoff {
   private currentBaseMs: number;
+  private timerPromise: CancelablePromise<void> | null = null;
 
   constructor(
+    /**
+     * The AsyncQueue to run backoff operations on.
+     */
+    private readonly queue: AsyncQueue,
+    /**
+     * The ID to use when scheduling backoff operations on the AsyncQueue.
+     */
+    private readonly timerId: TimerId,
```

Comment:
It would make more sense to me if this was the ID of the underlying operation. The fact that is is being backed-off seems secondary to me.